src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,28 @@ title = "Releases"
 weight = 30
 +++
 
+## v0.8.0
+
+* Added support for connecting clusters that use the OVNKubernetes CNI plugin.
```

Comment:
@nyechiel shall we include "Optimized route-agent driver"?

Basically, in the previous version of route-agent, we used to have a 30 secs refresh timer in endpoint listener (due to tech debt).
So, we used to receive endpoint (both local and remote) notifications periodically and as part of the handling, we used to flush out xfrm rules (aka cleanup handling on non-GW nodes) and validating if vx-submariner is present (if so, configuration is matching etc) and so on... every 30 seconds. 

Now, this is properly handled in the new driver and we do such checks only once during new notifications/transitions. So, the new route-agent will be optimal and should be consuming less CPU at the very least.

Reply:
We can mention this but I wouldn't go into much details.. the intention is to capture only user-facing changes here. @tpantelis wdyt?

Reply:
yeah this seems low-level technical and not really user-facing unless user(s) reported issues with high CPU, in which case we could state that the reported issue was fixed w/o going into the technical detail.